Resources/Prototypes/Entities/Objects/Materials/Sheets/glass.yml

```diff
@@ -80,6 +80,18 @@
   - type: Stack
     stackType: Glass
     count: 1
+    
+- type: entity
+  parent: SheetGlass
+  id: SheetGlass5
+  name: glass
+  suffix: Five
+  components:
+    - type: Sprite
+      state: glass
+    - type: Stack
+      stackType: Glass
+      count: 5
```

Comment:
These really shouldn't exist.

Reply:
I only added these as a "fast" way for the Ore processor to create sheets, instead of either 1 at a time or a full stack. A full stack would require 6 ore (since ore = 500 vol of material).
I'm open to alternative suggestions for this, like if there is a way to have a lathe produce multiple items at once?